Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
+
+function CardContent (props) {
```

Comment:
while a lot of these don't require much in the form of commenting I would just like to remind you that it is a good practice to add a comment to tell a viewer what this component is meant to do. Something simple like this component will render individual content for something and will be sent to <Parent component> can help some one find out what is going on when they first visit your code.